cartography/intel/crxcavator/crxcavator.py

```diff
@@ -49,55 +52,139 @@ def call_crxcavator_api(api_and_parameters, crxcavator_api_key, crxcavator_base_
     return data.json()
 
 
-def transform_extensions(extension_json):
+def get_extensions(crxcavator_api_key, crxcavator_base_url, extensions_list):
+    """
+    Retrieves the detailed information for all the extension_id and version pairs
+    :param crxcavator_api_key: The API key to access the CRXcavator service
+    :param crxcavator_base_url: The URL for the CRXcavator API
+    :param extensions_list: list of dictonary items containing the extension_id and version pairs
+    :return: list containing all metadata for extensions
+    """
+    extensions_details = []
+    for extension in extensions_list:
+        extension_id = extension.get('extension_id')
```

Comment:
should we be changing from `.get` to index referencing?  ex:  `extensions['extension_id']`.   

I'm under the assumption that extensions may not have an an `extension_id` if we're using `.get`